### PR TITLE
Update test results and add contribution guide

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# PEtab contribution guide
+
+We are happy about contributions to PEtab of any form.
+
+This is includes, but is not limited to:
```
Comment (yannikschaelte) [inline CONTRIBUTING.md]:
```suggestion
This includes, but is not limited to:
```

File: README.md [hunk 2]
```diff
@@ -41,51 +41,55 @@ A wide range of PEtab examples can be found in the systems biology parameter est
...
 
-  - [meigo](http://gingproc.iim.csic.es/meigo.html)
-
-  - [AMICI](https://github.com/ICB-DCM/AMICI/)
+  - [MEIGO](http://gingproc.iim.csic.es/meigo.html)
 
-  - [pyPESTO](https://github.com/ICB-DCM/pyPESTO/) ([Example](https://pypesto.readthedocs.io/en/latest/example/petab_import.html))
+  - [parPE](https://github.com/ICB-DCM/parPE/)
 
   - [pyABC](https://github.com/ICB-DCM/pyABC/)
```
Comment (yannikschaelte) [inline README.md]:
```suggestion
  - [pyABC](https://github.com/ICB-DCM/pyABC/) ([Example](https://pyabc.readthedocs.io/en/latest/examples/petab.html))
```

File: README.md [hunk 3]
```diff
@@ -41,51 +41,55 @@ A wide range of PEtab examples can be found in the systems biology parameter est
...
 
+  - [pyPESTO](https://github.com/ICB-DCM/pyPESTO/)
+    ([Example](https://pypesto.readthedocs.io/en/latest/example/petab_import.html))
+
 If your project or tool is using PEtab, and you would like to have it listed
-here, please let us know.
+here, please let us [know](https://github.com/PEtab-dev/PEtab/issues).
```
Comment (yannikschaelte) [inline README.md]:
```suggestion
here, please [let us know](https://github.com/PEtab-dev/PEtab/issues).
```

File: README.md [hunk 4]
```diff
@@ -41,51 +41,55 @@ A wide range of PEtab examples can be found in the systems biology parameter est
...
 
 ### PEtab features supported in different tools
 
 The following list provides an overview of supported PEtab features in
 different tools, based on passed test cases of the
 [PEtab test suite](https://github.com/PEtab-dev/petab_test_suite):
 
-| ID | Test                                                           | AMICI<br>*`develop`* | AMIGO | Copasi | D2D | dMod | pyPESTO<br>*`develop`* | pyABC
```
Comment (yannikschaelte) [inline README.md]: develop can be removed after today ...

Comment (dweindl) [reply README.md]: To be updated then.

**Pull request merged**